website/src/css/customTheme.scss

```diff
@@ -1103,12 +1103,11 @@ div[class^="tableOfContents"] {
     margin-left: -16px;
     padding-left: 12px;
     color: var(--ifm-font-color-base);
-    border-left: 4px solid var(--brand);
+    border-left: 4px solid var(--ifm-link-color);
 
     code {
-      font-weight: 600;
       color: var(--ifm-font-color-base);
-      white-space: nowrap;
```

Comment:
Let's roll back white space setting. It was added on purpose to prevent multiline monospace entries:

![Screenshot 2025-01-07 at 17 51 06](https://github.com/user-attachments/assets/77d46b24-418d-47b0-8a8f-e80e48a507f1)